modules/python-modules/syslogng/modules/kubernetes/scl/kubernetes.conf

```diff
@@ -170,6 +175,7 @@ block source kubernetes(prefix('.k8s.') base-dir("/var/log/containers") cluster-
         };
         parser {
             metrics-probe(
+                internal(yes)
```

Comment:
Why do we make this internal?

Reply:
It disables the parser level matched, not_matched counters. The newly generated ones remain :)

Reply:
Thanks.